gen3_util/files/manifest.py

```diff
@@ -122,13 +122,19 @@ def _write_indexd(index_client, project_id: str, manifest_item: dict, bucket_nam
             else:
                 logger.info(
                     f"NOT DELETING, MD5 didn't match existing record {manifest_item['object_id']} existing_record_md5:{existing_record['hashes']['md5']} manifest_md5:{manifest_item['md5']}")
+
+    authz = [f'/programs/{program}/projects/{project}']
+    if restricted_project_id:
```

Comment:
maybe make this a flag instead if this restricted project variable isn't ever going to be different than the 
normal PROJECT_ID env var since it would allow the user to specify 2 different program-project combinations and could lead to unintended outcomes